### DISCUSSION TITLE
Feature Request: display Revenue/Expense accounts

### DISCUSSION THREAD

**Discussion by Abejo** [victorbalssa/abacus#177] [Ideas]
Hi, I don't know if it's normal, but I only see one account, just like in the categories section.
In the graphics part I see well all the accounts I have

**Comment by victorbalssa** [comment]
Hey @Abejo,

You should see all assets accounts associated with the selected `currency` so if you have only 1 account with this currency it's normal.

You will need to to switch currency in order to access other accounts by pressing the filter button on top right of the screen (3 horizontal lines left to the arrow).

![image](https://github.com/victorbalssa/abacus/assets/12813321/c04b2a8d-fe19-4f97-a242-c9dface24952)

**Comment by Abejo** [comment]
Does it only show asset accounts? What I am not being able to see are the revenue accounts, don't these show them?

**Reply by victorbalssa** [reply]
Yes only `asset` accounts:
![image](https://github.com/victorbalssa/abacus/assets/12813321/21eca9cc-8505-43dc-b371-7c6d0fa1ef27)

I wanted to only shows the assets here, do you want to see `Revenue`/`Expense` accounts?
I can create a new tab on home.

**Comment by Abejo** [comment]
It would be great, or at least have the option to select which accounts you want to show.
